src/coreclr/nativeaot/Bootstrap/main.cpp

```diff
@@ -213,11 +212,7 @@ int main(int argc, char* argv[])
     if (initval != 0)
         return initval;
 
-    int retval = __managed__Main(argc, argv);
-
-    RhpShutdown();
```

Comment:
We only used `RhpShutdown` to detect explicit process exit and it was not good at that. (It would miss native code calling `ExitProcess()`, for example)

Reply:
Happy to see `RhpShutdown` gone!

Reply:
Cc @LakshanF on `RhpShutdown` going away. I think the plan was to do event pipe shutdown there. Will probably need a different place.